.github/workflows/auto-label.yml

```
@@ -0,0 +1,77 @@
+name: Auto Label Issues
+description: Automatically label issues based on keywords in title and body
```

actionlint (1.7.10)

[error] unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)